README.md

```diff
@@ -15,10 +15,24 @@ This fork primarily adds bug fixes and deprecation updates, to ensure that `shap
 
 We are eager to build a broad pool of maintainers, to avoid having a single person responsible for the entire repostitory. This repo adopts a _liberal contribution governance model_, where project decisions are based on a consensus seeking process. For more information, see [here](https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951).
 
+## Installation
+
+This fork is not yet available on PyPI or conda. Our goal is to merge the changes from this fork back into [slundberg/shap](https://github.com/slundberg/shap). If you would like to use this fork, the currently supported installation method is:
+
+```pip install git+https://github.com/dsgibbons/shap.git```
```

Comment:
````suggestion
```
pip install git+https://github.com/dsgibbons/shap.git
```
````

so it is rendered as a code block rather than inline.